server/controllers/auth/logout.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
```

Comment:
@ahmadysalah It can be helpful for debugging to always give your functions a name and then export them on a different line, instead of directly exporting an unnamed function.  For example:
`
const logout = (req, res) => {
  res.clearCookie(....
}
module.exports = logout;
```
Then, if you are debugging an exception, you would see that function name in the stack trace.

Reply:
Ok. I will change it.

Reply:
Aha, iam always ask my self why thats happening to me..
Its stupid missing 
Thanks again.